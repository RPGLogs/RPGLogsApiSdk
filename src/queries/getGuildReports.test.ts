import { buildTestSdk } from '../testHelpers';

test('returns data', async () => {
  const response = await buildTestSdk().getGuildReports({
    guildName: 'Reaction',
    guildServerSlug: 'Kazzak',
    guildServerRegion: 'EU',
    limit: 1,
    zoneId: 24,
    includeFights: true,
  });

  expect(response?.reportData?.reports).toMatchSnapshot();
});
