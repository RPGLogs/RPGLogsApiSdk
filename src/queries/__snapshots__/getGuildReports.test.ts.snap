// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns data 1`] = `
Object {
  "current_page": 1,
  "data": Array [
    Object {
      "code": "TLbM9tGKj162JAyp",
      "endTime": 1603137864064,
      "exportedSegments": 53,
      "fights": Array [
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2329,
          "endTime": 747281,
          "fightPercentage": 0,
          "id": 1,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 1,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Wrathion",
          "rating": null,
          "size": 10,
          "startTime": 700368,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 781962,
          "fightPercentage": null,
          "id": 2,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Monstrous Behemoth",
          "rating": null,
          "size": null,
          "startTime": 765210,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 850538,
          "fightPercentage": null,
          "id": 3,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Anubisath Sentinel",
          "rating": null,
          "size": null,
          "startTime": 793984,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 915023,
          "fightPercentage": null,
          "id": 4,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Anubisath Sentinel",
          "rating": null,
          "size": null,
          "startTime": 884833,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 991150,
          "fightPercentage": null,
          "id": 5,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Anubisath Sentinel",
          "rating": null,
          "size": null,
          "startTime": 950659,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 133.3333282470703,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1080126,
          "fightPercentage": null,
          "id": 6,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Disciple of the Prophet",
          "rating": null,
          "size": null,
          "startTime": 1080126,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2334,
          "endTime": 1222467,
          "fightPercentage": 0,
          "id": 7,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Prophet Skitra",
          "rating": null,
          "size": 10,
          "startTime": 1097908,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.44444274902344,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1272884,
          "fightPercentage": null,
          "id": 8,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Occult Shadowmender",
          "rating": null,
          "size": null,
          "startTime": 1261908,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.44444274902344,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1310218,
          "fightPercentage": null,
          "id": 9,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Monstrous Behemoth",
          "rating": null,
          "size": null,
          "startTime": 1287846,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.44444274902344,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1351392,
          "fightPercentage": null,
          "id": 10,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Ossirat",
          "rating": null,
          "size": null,
          "startTime": 1320256,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1433286,
          "fightPercentage": null,
          "id": 11,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Aqir Shadowcrafter",
          "rating": null,
          "size": null,
          "startTime": 1373371,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2327,
          "endTime": 1781683,
          "fightPercentage": 0,
          "id": 12,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 1,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Maut",
          "rating": null,
          "size": 10,
          "startTime": 1595531,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.39999389648438,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1818588,
          "fightPercentage": null,
          "id": 13,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Onyx Scarab",
          "rating": null,
          "size": null,
          "startTime": 1818547,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 132.3333282470703,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1863123,
          "fightPercentage": null,
          "id": 14,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Maddened Conscript",
          "rating": null,
          "size": null,
          "startTime": 1837657,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 132.3333282470703,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 1992163,
          "fightPercentage": null,
          "id": 15,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Umbral Gatekeeper",
          "rating": null,
          "size": null,
          "startTime": 1914677,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 2167009,
          "fightPercentage": null,
          "id": 16,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Jesh'ra",
          "rating": null,
          "size": null,
          "startTime": 2007525,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 2232592,
          "fightPercentage": null,
          "id": 17,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Monstrous Behemoth",
          "rating": null,
          "size": null,
          "startTime": 2178923,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 128.5,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 2290629,
          "fightPercentage": null,
          "id": 18,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Queen Azshara",
          "rating": null,
          "size": null,
          "startTime": 2284991,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2328,
          "endTime": 2901316,
          "fightPercentage": 0,
          "id": 19,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Dark Inquisitor Xanesh",
          "rating": null,
          "size": 10,
          "startTime": 2294282,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.7142791748047,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 2904714,
          "fightPercentage": null,
          "id": 20,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Queen Azshara",
          "rating": null,
          "size": null,
          "startTime": 2904714,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 3041156,
          "fightPercentage": null,
          "id": 21,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Vexiona",
          "rating": null,
          "size": null,
          "startTime": 2953075,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.5,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2336,
          "endTime": 3351372,
          "fightPercentage": 0,
          "id": 22,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 1,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Vexiona",
          "rating": null,
          "size": 10,
          "startTime": 3159061,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 3612051,
          "fightPercentage": null,
          "id": 23,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Drest'agath",
          "rating": null,
          "size": null,
          "startTime": 3387874,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 128,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 3645124,
          "fightPercentage": null,
          "id": 24,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Living Blood",
          "rating": null,
          "size": null,
          "startTime": 3641221,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2343,
          "endTime": 3767311,
          "fightPercentage": 0,
          "id": 25,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Drest'agath",
          "rating": null,
          "size": 10,
          "startTime": 3655096,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 3823293,
          "fightPercentage": null,
          "id": 26,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Leeching Parasite",
          "rating": null,
          "size": null,
          "startTime": 3793861,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 3840356,
          "fightPercentage": null,
          "id": 27,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Eye of the Depths",
          "rating": null,
          "size": null,
          "startTime": 3838371,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 3883364,
          "fightPercentage": null,
          "id": 28,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Eye of the Depths",
          "rating": null,
          "size": null,
          "startTime": 3858864,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2335,
          "endTime": 4027747,
          "fightPercentage": 0,
          "id": 29,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 3,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Shad'har the Insatiable",
          "rating": null,
          "size": 10,
          "startTime": 3921659,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 4101859,
          "fightPercentage": null,
          "id": 30,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Corrosive Digester",
          "rating": null,
          "size": null,
          "startTime": 4041348,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": 93.02,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2345,
          "endTime": 4319985,
          "fightPercentage": 93.02,
          "id": 31,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": false,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Il'gynoth, Corruption Reborn",
          "rating": null,
          "size": 10,
          "startTime": 4151369,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 4371259,
          "fightPercentage": null,
          "id": 32,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Gibbering Ooze",
          "rating": null,
          "size": null,
          "startTime": 4371259,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0.38,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2345,
          "endTime": 4739109,
          "fightPercentage": 0.38,
          "id": 33,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Il'gynoth, Corruption Reborn",
          "rating": null,
          "size": 10,
          "startTime": 4392922,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.11111450195312,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 4789443,
          "fightPercentage": null,
          "id": 34,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Umbral Seer",
          "rating": null,
          "size": null,
          "startTime": 4757810,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.55555725097656,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 4900986,
          "fightPercentage": null,
          "id": 35,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Umbral Seer",
          "rating": null,
          "size": null,
          "startTime": 4819328,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.42857360839844,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 4959385,
          "fightPercentage": null,
          "id": 36,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Aqir Crusher",
          "rating": null,
          "size": null,
          "startTime": 4919005,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2333,
          "endTime": 5115901,
          "fightPercentage": 0,
          "id": 37,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "The Hivemind",
          "rating": null,
          "size": 10,
          "startTime": 5034947,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 133.75,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 5134624,
          "fightPercentage": null,
          "id": 38,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Engorged Maggot",
          "rating": null,
          "size": null,
          "startTime": 5134624,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 5178356,
          "fightPercentage": null,
          "id": 39,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Abyssal Watcher",
          "rating": null,
          "size": null,
          "startTime": 5154647,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 5188672,
          "fightPercentage": null,
          "id": 40,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Ra-den the Despoiled",
          "rating": null,
          "size": null,
          "startTime": 5178426,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 5235909,
          "fightPercentage": null,
          "id": 41,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Abyssal Watcher",
          "rating": null,
          "size": null,
          "startTime": 5188705,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 5244343,
          "fightPercentage": null,
          "id": 42,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Ra-den the Despoiled",
          "rating": null,
          "size": null,
          "startTime": 5235939,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2331,
          "endTime": 5551413,
          "fightPercentage": 0,
          "id": 43,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 2,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Ra-den the Despoiled",
          "rating": null,
          "size": 10,
          "startTime": 5344361,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 5680187,
          "fightPercentage": null,
          "id": 44,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Monstrous Behemoth",
          "rating": null,
          "size": null,
          "startTime": 5595613,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 5790489,
          "fightPercentage": null,
          "id": 45,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Amalgamation of Flesh",
          "rating": null,
          "size": null,
          "startTime": 5709335,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.3000030517578,
          "bossPercentage": 65.58,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2337,
          "endTime": 6199312,
          "fightPercentage": 65.58,
          "id": 46,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": false,
          "lastPhase": 3,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Carapace of N'Zoth",
          "rating": null,
          "size": 10,
          "startTime": 5886429,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 129.8333282470703,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 6245806,
          "fightPercentage": null,
          "id": 47,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Unknown",
          "rating": null,
          "size": null,
          "startTime": 6244383,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 37.56,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2337,
          "endTime": 6549519,
          "fightPercentage": 37.56,
          "id": 48,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": false,
          "lastPhase": 3,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Carapace of N'Zoth",
          "rating": null,
          "size": 10,
          "startTime": 6280122,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 36.35,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2337,
          "endTime": 6817397,
          "fightPercentage": 36.35,
          "id": 49,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": false,
          "lastPhase": 3,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Carapace of N'Zoth",
          "rating": null,
          "size": 10,
          "startTime": 6603189,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2337,
          "endTime": 7211492,
          "fightPercentage": 0,
          "id": 50,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 3,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "Carapace of N'Zoth",
          "rating": null,
          "size": 10,
          "startTime": 6927144,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.10000610351562,
          "bossPercentage": 0,
          "completeRaid": false,
          "difficulty": 3,
          "encounterID": 2344,
          "endTime": 7731791,
          "fightPercentage": 0,
          "id": 51,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": true,
          "lastPhase": 2,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "N'Zoth the Corruptor",
          "rating": null,
          "size": 10,
          "startTime": 7307734,
          "wipeCalledTime": null,
        },
        Object {
          "averageItemLevel": 131.39999389648438,
          "bossPercentage": null,
          "completeRaid": false,
          "difficulty": null,
          "encounterID": 0,
          "endTime": 7735586,
          "fightPercentage": null,
          "id": 52,
          "keystoneAffixes": null,
          "keystoneBonus": null,
          "keystoneLevel": null,
          "keystoneTime": null,
          "kill": null,
          "lastPhase": 0,
          "lastPhaseIsIntermission": false,
          "layer": null,
          "name": "N'Zoth the Corruptor",
          "rating": null,
          "size": null,
          "startTime": 7731819,
          "wipeCalledTime": null,
        },
      ],
      "guildTag": null,
      "owner": Object {
        "id": 381152,
        "name": "Fluclox",
      },
      "revision": 53,
      "segments": 53,
      "startTime": 1603130114982,
      "title": "Glory of the Nyalotha Raider",
      "visibility": "public",
      "zone": Object {
        "id": 24,
        "name": "Ny'alotha",
      },
    },
  ],
  "from": 1,
  "has_more_pages": true,
  "last_page": 132,
  "per_page": 1,
  "to": 1,
  "total": 132,
}
`;
